package.json

```diff
@@ -19,10 +19,10 @@
   ],
   "dependencies": {
     "aes-js": "^3.1.2",
-    "bfx-api-node-core": "^1.5.5",
+    "bfx-api-node-core": "git+https://github.com/avsek477/bfx-api-node-core.git#orders-models-update",
     "bfx-api-node-plugin-wd": "^1.0.4",
     "bfx-api-node-rest": "^4.3.1",
-    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.7.3",
+    "bfx-hf-algo": "git+https://github.com/avsek477/bfx-hf-algo.git#add-visible-on-hit-for-algo-order",
```

Comment:
needs to be changed after this [PR](https://github.com/bitfinexcom/bfx-hf-algo/pull/194) is merged.

Reply:
```suggestion
    "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.10.0",
```